AnnoyancesFilter/sections/self-promo.txt

```diff
@@ -1376,6 +1376,7 @@ kurkul.com##.text_news > div.widget_image-vertical
 pravda.com.ua##.unit_level_banner
 pravda.com.ua##div[class^="club_banner"]
 beauty.rakuten.co.jp##.SideColumn__banner
+beauty.rakuten.co.jp###footerPr
```

Comment:
Please move to mobile. It is blocked by not optimized filter and I fixed it
```
!+ NOT_OPTIMIZED
_ad_content.
```

![image](https://user-images.githubusercontent.com/8361299/172703216-4041e077-bb0c-4035-acfd-e265499e72a3.png)

Reply:
done